connectors/src/connectors/intercom/index.ts

```diff
@@ -220,12 +221,49 @@ export async function cleanupIntercomConnector(
     return new Err(new Error("Connector not found"));
   }
 
+  try {
+    const accessToken = await getAccessTokenFromNango({
```

Comment:
I will move that in intercom_api after your merge so that all calls are centralized and we share the logic to get the access token and fix the same API version for all calls 👍🏻

Reply:
ack